src/amcl-extensions/big_256_56.h

```diff
@@ -24,9 +24,10 @@
 extern "C" {
 #endif
 
+struct ecdaa_prng;
```

Comment:
That started with a circular dependency I had in an early version of this project, and I kept doing it just as a nod to compilation time reduction. But, this is a tiny project, so that's really not an issue. I can change it easily if you think it confuses things.

Reply:
Haha, my reason for importing headers was going to be that "it helps ensure circular dependencies don't creep in".

I think eventually standard imports will be better, but that's an easy change to make down the line.